src/celeritas/phys/PhysicsTrackView.hh

```diff
@@ -60,6 +60,9 @@ class PhysicsTrackView
     // Reset the remaining MFP to interaction
     inline CELER_FUNCTION void reset_interaction_mfp();
 
+    // Set the energy loss range for the current material and particle energy
+    inline CELER_FUNCTION void dedx_range(real_type);
```

Comment:
I added a `PhysicsStepView` a few refactors ago because this class was getting big... the "physics step view" is meant for temporary data that's valid only within a step: cached data like the range is a good example of that. Could you move these accessors to that class?

Reply:
Okay I overlooked PhysicsStepView as all UrbanMsc related classes currently use PhysicsTrackView.  There is no problem to move them to PhysicsStepView and replace PhysicsTrackView by PhysicsStepView in all UrbanMsc related interfaces and tests.  Since there are a number of places where we have to change interfaces and construct PhysicsStepView accordingly, can we do it in a separate MR after this MR?

Reply:
That would be a bit of work so we can do it in a separate PR. Thanks!